includes/embeds/class-amp-gallery-embed.php

```diff
@@ -62,6 +62,7 @@ public function shortcode( $attr ) {
 				'order'          => $atts['order'],
 				'orderby'        => $atts['orderby'],
 				'fields'         => 'ids',
+				'posts_per_page' => 100,
```

Comment:
Note the lack of `posts_per_page` here and below was to align the queries here with the corresponding queries in core's `gallery_shortcode()` function: https://github.com/WordPress/wordpress-develop/blob/79bf20d320799e6c1d9d8b32b63d4fcea0910a75/src/wp-includes/media.php#L1746-L1768

I highly doubt that there will be more than 100 images in a gallery, but if there were, then the AMP version of the gallery would be truncated with this change.

Reply:
Okay, good point, I'm going to roll this one back then. I don't think we've ever seen issues with the core code so we should be safe.